gax/src/main/java/com/google/api/gax/tracing/ApiTracerFactory.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2017 Google LLC
```

Comment:
Please correct year here and in other new files of this PR. At least this one is definitely wrong. As for 2018 vs 2019, we can keep the year of creation (i.e. 2018), though I feel the year when it gets into master is the proper one (i.e. 2019)... Also it kind of gives the privilege to submit first 2019 headers in the repo =).

Reply:
done